### PR TITLE
Update HOWTORELEASE based on the 3.0.0 release

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: HOWTORELEASE.rst [hunk 1]
```diff
@@ -3,90 +3,83 @@ How to release pytest
...
-       ln -s 5.4 latest
+11. Send release announcement to mailing lists:
 
-    Browse to pytest.org to verify.
+    - pytest-dev@python.org
+    - testing-in-python@lists.idyll.org
+    - python-announce-list@python.org
 
-11. Publish to pypi::
+    And aanounce the release on Twitter, making sure to add the hashtag ``#pytest``.
```
Comment (The-Compiler) [inline HOWTORELEASE.rst]: aanounce -> announce

File: HOWTORELEASE.rst [hunk 2]
```diff
@@ -3,90 +3,83 @@ How to release pytest
...
-1. Bump version numbers in _pytest/__init__.py (setup.py reads it)
+2. Check and finalize ``CHANGELOG.rst``.
 
-2. Check and finalize CHANGELOG
+3. Write ``doc/en/announce/release-VERSION.txt`` and include
+   it in ``doc/en/announce/index.txt``. Run this command to list names of authors involved::
 
-3. Write doc/en/announce/release-VERSION.txt and include
-   it in doc/en/announce/index.txt::
+        git log 2.8.2..HEAD --format='%aN' | sort -u
```
Comment (blueyed) [inline HOWTORELEASE.rst]: `git log $(git tag --sort=version:refname | tail -n1)..HEAD --format='%aN' | sort -u` to make it use the last tag automatically?

File: HOWTORELEASE.rst [hunk 3]
```diff
@@ -3,90 +3,83 @@ How to release pytest
...
-      cd doc/en
-      make html
-
-   Commit any changes before tagging the release.
-
-9. Tag the release::
-
-      git tag VERSION
-      git push
+      devpi push pytest==VERSION pypi:NAME
```
Comment (blueyed) [inline HOWTORELEASE.rst]: What is `NAME` here?  Isn't it `pytest` always?

File: HOWTORELEASE.rst (hunk 2 above)
Comment (The-Compiler) [inline HOWTORELEASE.rst]: `git describe --abbrev=0 --tags` should do the same you did in `$()` there

**Comment by blueyed** [comment]
Nice improvements!

File: HOWTORELEASE.rst (hunk 1 above)
Comment (nicoddemus) [inline HOWTORELEASE.rst]: Fixed, thanks!

**Pull request merged**